src/rgw/rgw_admin.cc

```diff
@@ -56,7 +56,6 @@ extern "C" {
 #include "rgw_http_client_curl.h"
 #include "rgw_zone.h"
 #include "rgw_pubsub.h"
-#include "rgw_sync_module_pubsub.h"
```

Comment:
should keep this line. otherwise the prototype of `RGWPSSyncModuleInstance` is missing in this file.


```
/home/jenkins-build/build/workspace/ceph-dev-new-build/ARCH/x86_64/AVAILABLE_ARCH/x86_64/AVAILABLE_DIST/centos7/DIST/centos7/MACHINE_SIZE/gigantic/release/15.2.10-265-g61ce6234/rpm/el7/BUILD/ceph-15.2.10-265-g61ce6234/src/rgw/rgw_admin.cc:9249:33: error: 'RGWPSSyncModuleInstance' does not name a type; did you mean 'RGWSyncModuleInstance'?
     auto psmodule = static_cast<RGWPSSyncModuleInstance *>(store->getRados()->get_sync_module().get());
                                 ^~~~~~~~~~~~~~~~~~~~~~~
                                 RGWSyncModuleInstance
```